Makefile

```diff
@@ -62,7 +62,12 @@ verify-boilerplate:
 test-unit:
 	go test -v ./cmd/... ./pkg/...
 
-test: test-unit test-wordpress
+test-integration:
```

Comment:
Should this depend on `deploy`? Or something that tests it has been deployed?

Reply:
To me, that sounds too much at this point.